exercises/practice/hello-world/src/lib.cairo

```diff
@@ -0,0 +1,7 @@
+// ByteArray is a string that's not limited to 31 characters, you'll learn more about this later
+fn hello() -> ByteArray {
+    "Goodbye, Mars!"
+}
+
+#[cfg(test)]
+mod tests;
```

Comment:
I don't see this bit in the example implementation. Should that also contain this part?